### PR TITLE
autotest.sh with docker support for postgres

### PR DESCRIPTION
**Pull request by MorrisJobke**
- waits until the docker container is started
- `export USEDOCKER=1` to run the docker version of postgres

Now there is now need to have a local postgres instance anymore. No excuse to run no Postgres unit test :speak_no_evil: 

cc @deepdiver1975 @nickvergessen @PVince81

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
Jenkins doesn't like this

**Comment by PVince81** [comment]
I ran this locally and it worked nicely.

@MorrisJobke can you rebase ?

**Comment by MorrisJobke** [comment]
> @MorrisJobke can you rebase ?

Worked for me locally too, but not on jenkins :(

**Comment by MorrisJobke** [comment]
Rebased and re-run locally. All works fine.

File: autotest.sh [hunk 1]
```diff
@@ -177,6 +190,13 @@ function execute_tests {
 		"${PHPUNIT[@]}" --configuration phpunit-autotest.xml --log-junit "autotest-results-$1.xml" "$2" "$3"
 		RESULT=$?
 	fi
+	exit;
```
Comment (MorrisJobke) [inline autotest.sh]: Weird thing. I can see the output from below, but how can this happen if the script exits here. I will test this.

Comment (MorrisJobke) [inline autotest.sh]: Got it ...debug stuff. I will remove it once Jenkins finished.

**Comment by MorrisJobke** [comment]
@DeepDiver1975 @nickvergessen @rullzer @PVince81 Time for review :smiley:

**Comment by PVince81** [comment]
:+1: works

**Comment by MorrisJobke** [comment]
@DeepDiver1975 ;)

**Comment by DeepDiver1975** [comment]
let me test .....

**Comment by DeepDiver1975** [comment]
works :+1:

**Pull request merged**